code/__DEFINES/reagents.dm

```diff
@@ -29,6 +29,8 @@
 #define SEALED_CONTAINER (1<<10)
 // Is an open container for all intents and purposes.
 #define OPENCONTAINER (REFILLABLE | DRAINABLE | TRANSPARENT)
+//Prevents spilling and splashing but does prevent pouring and drinking reagents like the badly named spillable var.
+#define SMART_CAP (1<<11)
```

Comment:
this should be above `OPENCONTAINER` (which should have an _ in its name ftr but that's not your problem)
also put a space before the text to be internally consistent

Reply:
ok